packages/netlify-cms-widget-markdown/src/MarkdownControl/index.js

```diff
@@ -34,9 +36,12 @@ export default class MarkdownControl extends React.Component {
   constructor(props) {
     super(props);
     editorControl = props.editorControl;
+    allowedModes = [...(props.field.get('mode') ?? ['raw', 'rich_text'])];
```

Comment:
Using a global var is discouraged.
How about having a class method:
```js
getAllowedMode()
```
That way we always get the up to date value, even when the component props change.

Also you can use:
```js
import { List } from 'immutable';
...

props.field.get('mode', List(['raw', 'rich_text'])).toArray()
```

Reply:
Agreed. Done.